docs/ops/activation/SoftPlus_4.md

```diff
@@ -13,9 +13,23 @@
 *SoftPlus* performs element-wise activation function on a given input tensor, based on the following mathematical formula:
 
 \f[
-SoftPlus(x) = \ln(1+e^{x})
+SoftPlus(x) = \left\{\begin{array}{r}
+    x \qquad \mbox{if } x \geq threshold \\
+    log(e^{x} + 1.0) \qquad \mbox{if } x < threshold
+\end{array}\right.
 \f]
 
+**Note**: For numerical stability the operation reverts the linear function when `x > threshold` where `threshold` depends on *T* and
+is chosen in such a way that the difference between the linear function and exact calculation is no more than `1e-6`.
+The `threshold` can be calculated with the following formula where `alpha` is the count of the numbers after the decimal point,
```

Comment:
`count of the numbers` -> `number of digits`

Reply:
Thanks, done!